src/ol/Graticule.js

```diff
@@ -71,6 +71,9 @@ const INTERVALS = [
  * @property {number} [latLabelPosition=1] Latitude label position in fractions
  * (0..1) of view extent. 0 means at the left of the viewport, 1 means at the
  * right.
+ * @property {function(module:ol/extent~Extent):module:ol/extent~Extent} [getExtent] An optional function
+ * that returns the extent where the graticule will be displayed.
+ * The function receives the view extent as argument.
```

Comment:
It doesn't look like it does, and I think it's not needed.

Reply:
If users modify the passed extent in place, things will break wherever the library relies on `frameState.extent`.